recipe/meta.yaml

```
@@ -35,6 +35,7 @@ outputs:
       ignore_run_exports_from:
         - {{ compiler("cuda") }}                 # [cuda_compiler_version != "None"]
         - libarrow-all
+        - numpy
```

Could we write a comment here like
```suggestion
        # we don't need numpy at runtime, just to build
        - numpy
```
(I haven't actually understood what happens in arrow there, so would appreciate you coming up with a formulation).

---

That's essentially it yes: it is not required at run-time (you can import pyarrow and use functionality that doesn't require numpy), but we still always build the numpy-related functionality (we are using the numpy C API), so it is still a required build dependency.